catalog/imaging/imaging-transformer-nitf/src/main/java/org/codice/alliance/transformer/nitf/gmti/AcftbAttribute.java

```diff
@@ -14,51 +14,106 @@
+     * appropriate
+     */
 
-enum AcftbAttribute implements NitfAttribute<Tre> {
     AIRCRAFT_MISSION_ID(Isr.MISSION_ID,
             "AC_MSN_ID",
             tre -> GmtiTreUtility.getTreValue(tre, "AC_MSN_ID"),
-            new IsrAttributes()),
+            new IsrAttributes().getAttributeDescriptor(Isr.MISSION_ID),
+            "aircraftMissionId"),
     AIRCRAFT_TAIL_NUMBER(Isr.PLATFORM_ID,
             "AC_TAIL_NO",
             tre -> GmtiTreUtility.getTreValue(tre, "AC_TAIL_NO"),
-            new IsrAttributes()),
+            new IsrAttributes().getAttributeDescriptor(Isr.PLATFORM_ID),
+            "aircraftTailNumber"),
     SENSOR_ID_TYPE(Isr.SENSOR_TYPE,
             "SENSOR_ID_TYPE",
             tre -> GmtiTreUtility.getTreValue(tre, "SENSOR_ID_TYPE"),
-            new IsrAttributes()),
+            new IsrAttributes().getAttributeDescriptor(Isr.SENSOR_TYPE),
+            "sensorIdType"),
     SENSOR_ID(Isr.SENSOR_ID,
             "SENSOR_ID",
             tre -> GmtiTreUtility.getTreValue(tre, "SENSOR_ID"),
-            new IsrAttributes());
+            new IsrAttributes().getAttributeDescriptor(Isr.SENSOR_ID),
+            "sensorId"),
+
+    /*
+     * Non-normalized attributes
+     */
+
+    AIRCRAFT_TAKEOFF("aircraftTakeOff", "AC_TO", tre -> GmtiTreUtility.getTreValue(tre, "AC_TO")),
+    SCENE_SOURCE("sceneSource",
+            "SCENE_SOURCE",
+            tre -> GmtiTreUtility.getTreValue(tre, "SCENE_SOURCE")),
+    SCENE_NUMBER("sceneNumber", "SCNUM", tre -> GmtiTreUtility.getTreValue(tre, "SCNUM")),
+    PROCESSING_DATE("processingDate", "PDATE", tre -> GmtiTreUtility.getTreValue(tre, "PDATE")),
+    IMMEDIATE_SCENE_HOST("immediateSceneHost",
+            "IMHOSTNO",
+            tre -> GmtiTreUtility.getTreValue(tre, "IMHOSTNO")),
+    IMMEDIATE_SCENE_REQUEST_ID("immediateSceneRequestId",
+            "IMREQID",
+            tre -> GmtiTreUtility.getTreValue(tre, "IMREQID")),
+    MISSION_PLAN_MODE("missionPlanMode", "MPLAN", tre -> GmtiTreUtility.getTreValue(tre, "MPLAN"));
+
+    private static final String ATTRIBUTE_NAME_PREFIX = "acftb.";
 
     private String shortName;
 
     private String longName;
 
     private Function<Tre, Serializable> accessorFunction;
 
-    private AttributeDescriptor attributeDescriptor;
+    private Set<AttributeDescriptor> attributeDescriptors;
+
+    AcftbAttribute(final String longName, final String shortName,
+            final Function<Tre, Serializable> accessorFunction) {
+        this.longName = longName;
+        this.shortName = shortName;
+        this.accessorFunction = accessorFunction;
+        // retrieving metacard attribute descriptors for this attribute to prevent later lookups
+        this.attributeDescriptors = new HashSet<>();
+        this.attributeDescriptors.add(new AttributeDescriptorImpl(
+                ExtNitfUtility.EXT_NITF_PREFIX + ATTRIBUTE_NAME_PREFIX + longName,
+                true,
```

Comment:
It might be helpful to throw a comment on these booleans for the AttributeDescriptorImpls in this PR so someone can know what they are without referencing the constructor.  We do this in CSW :  https://github.com/bdeining/ddf/blob/master/catalog/spatial/csw/spatial-csw-common/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/common/CswRecordMetacardType.java#L535.